coalib/bears/requirements/PackageRequirement.py

```diff
@@ -0,0 +1,28 @@
+from coala_decorators.decorators import generate_repr
+
+
+@generate_repr()
+class PackageRequirement:
+
```

Comment:
ah actually i might be wrong, it might be unneeded, but i think i applied coala and it happened, i dont bet